ArduCopter/GCS_Mavlink.cpp

```diff
@@ -696,6 +697,7 @@ void GCS_MAVLINK_Copter::packetReceived(const mavlink_status_t &status,
         // optional handling of GLOBAL_POSITION_INT as a MAVLink based avoidance source
         copter.avoidance_adsb.handle_msg(msg);
     }
+    copter.mode_chase.mavlink_packet_received(msg);
```

Comment:
to be consistent with other similar calls we should probably call this "handle_msg".